App.js

```diff
@@ -15,6 +16,7 @@ import GoalInput from "./components/GoalInput";
 
 export default function App() {
   const [courseGoals, setCourseGoals] = useState([]);
+
   const [modalIsVisible, setModalIsVisible] = useState(false);
```

Comment:
you can rename the state 
```suggestion
  const [isModalVisible, setIsModalVisible] = useState(false);
```